core/ir/aarch64/encode.c

```diff
@@ -44,42 +44,42 @@
 
```

Comment:
Grammar errors: s/lacked/missing/ in the title and description

Reply:
Fixed

Reply:
> JerryYouxin changed the title from `Fix lacked reg encoding info for aarch64` to `Fixs lacked missing reg encoding info for aarch64`

Maybe you're not familiar with the UNIX sed/perl/bash syntax `s/replace-this/with-this/`?  The ungrammatical part is `lacked` and it should be replaced with `missing`.  We now also have another grammar error `Fixs`.

Reply:
I will fix the title and description.